members/WIL000477.yaml

```diff
@@ -1,50 +1,37 @@
 bioguide: WIL000477
 contact_form:
-  method: post
-  action: ""
   steps:
-    - visit: "https://legis.wisconsin.gov/assembly/05/steineke/contact/"
+    - visit: "https://legis.wisconsin.gov/assembly/05/steineke/contact-form/"
     - find:
         - selector: input[type='radio'][value='No']
-    - click_on:
-        - value: newsletter
-          selector: input[type='radio'][value='No']
-    - wait:
-        - value: 1
     - fill_in:
         - name: Name
           selector: div .name input
           value: $NAME_FULL
           required: true
         - name: Address
-          selector: div .streetaddress input
+          selector: div .address input
           value: $ADDRESS_STREET
           required: true
         - name: City
           selector: div .city input
           value: $ADDRESS_CITY
           required: true
         - name: email
-          selector: div .email input
+          selector: div .emailorphonenumber input
```

Comment:
lol this is such a weird idea for a form.. combining phone and email??

Reply:
yeah, tell me about it :sigh: